modules/messages/src/inbound_lane.rs

```diff
@@ -136,9 +136,7 @@ impl<S: InboundLaneStorage> InboundLane<S> {
 			return ReceivalResult::TooManyUnconfirmedMessages;
 		}
 
-		// dispatch message before updating anything in the storage. If dispatch would panic,
-		// (which should not happen in the runtime) then we simply won't consider message as
-		// delivered (no changes to the inbound lane storage have been made).
+		// then, dispatch message using pallet-bridge-dispatch
```

Comment:
Messages pallet may work with any kind of dispatch mechanisms. E.g. in future we may switch to XCM (#214). So I'd prefer to keep comment more generic :)

Reply:
That makes sense. Updated